flink-python/docs/reference/pyflink.table/catalog.rst

```diff
@@ -250,3 +250,31 @@ A CatalogDescriptor is a template for creating a catalog instance. It closely re
     :toctree: api/
 
     CatalogDescriptor.of
+
+
+ObjectIdentifier
+----------------
+
+Identifies an object in a catalog. It allows to identify objects such as tables, views,
```

Comment:
nit: `It allows to identify objects ` does not read well. I suggest
`Identifies an object in a catalog, including tables, views, functions and types.`